fe/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -135,23 +138,23 @@ public long createLoadJobFromMiniLoad(TMiniLoadBeginRequest request) throws User
         LoadJob loadJob = null;
         writeLock();
         try {
-            checkLabelUsed(database.getId(), request.getLabel(), request.getRequest_id());
+            // checkLabelUsed(database.getId(), request.getLabel(), request.getRequest_id());
```

Comment:
Why comment this check?  better to remove thie code

Reply:
removed